src/actions/websocket.js

```diff
@@ -381,16 +381,7 @@ function handleNewPostEvent(msg) {
                 dispatch(makeGroupMessageVisibleIfNecessary(post.channel_id));
             }
 
-            actions.push({
-                type: PostTypes.RECEIVED_POSTS,
-                data: {
-                    order: [],
-                    posts: {
-                        [post.id]: post,
-                    },
-                },
-                channelId: post.channel_id,
-            });
+            actions.push(receivedNewPost(post));
```

Comment:
nothing to do here, but this will conflict with #763

Reply:
Yeah, this PR also conflicts with https://github.com/mattermost/mattermost-redux/pull/735 as well. This is on the branch, so we can deal with those once this branch is updated